cves/kernel/CVE-2014-7283.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The xfs_da3_fixhashpath function in the xfs implementation in the Linux 
```

Comment:
Could this be written in simpler language for readers who are not familiar with the kernel?

Reply:
Good call, I have simplified the language to be less specific to the Linux kernel.